xbstrap/__init__.py

```diff
@@ -77,6 +77,7 @@ def do_runtool(args):
         tool_pkgs=tool_pkgs,
         workdir=workdir,
         for_package=for_package,
+        containerless=tool_pkgs[0].containerless or False,
```

Comment:
Why do we check `tool_pkgs[0]` here?

Reply:
This was a testing hack, which has now been replaced with a functional-style fold over the tool list.